Jh press schema changes

JonHall101 opened the pull request:
Feedback from populating the press schema with data from the 1m press raised some slight changes to requirements. Press schema now includes an array of heaters to avoid the need for different devices using the same connection. Tool schema now includes arrays of pressure and temperature tags rather than single tags to allow for multiple sensors on a single tool.

JonHall101 committed:
Update Press-v1.json

added an array of heaters under press. using Heater-v1 schema

JonHall101 committed:
Update Tool-v1.json

temperature and pressure tags are expanded to arrays

amrc-benmorrow commented on Press/Press-v1.json:
```diff
@@ -63,7 +67,9 @@
-      ]
+      ],
+      "uuid": "a9d2f476-25fe-49ba-a1d4-a70ea48c756a",
+      "index": 0
```
Something has gone wrong here... were these exported from the Schema Editor?

JonHall101 replied:
yes, I've spotted that now. I'm just raising a new PR

amrc-benmorrow commented on Press/Tool-v1.json:
```diff
@@ -30,11 +30,21 @@
         }
       ]
     },
-    "Pressure": {
```
Strictly speaking this is an incompatible change (a metric has been removed). Do we need some way to mark schemas as 'under development'?

JonHall101 replied:
does it change anything that I am the only person using this (i appreciate this isn't best practice)

amrc-benmorrow replied:
I think it should change things; but with these published publically we don't know who is or isn't using what. For now I think we don't worry, but we should either choose some way of marking schemas 'experimental: subject to breaking changes' or find a way to develop them privately.

JonHall101 committed:
Update Press-v1.json

schema exported from acs manager did not match the original schema. changes have been made to acomodate.

JonHall101 committed:
Update Press-v1.json

removed tools schema uuid

JonHall101 committed:
Update Press-v1.json

JSON Validated

JonHall101 committed:
Update Press-v1.json

removed uuid
